apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -1990,6 +2012,10 @@ else:
         });
 
         $("#fileUploader").on('fb:updatePath', function (e, options) {
+          const uploadingToOzone = self.currentPath().startsWith("ofs://");          
+          if (uploadingToOzone && UPLOAD_CHUNK_SIZE) {
+            uploader.setOption('sizeLimit', UPLOAD_CHUNK_SIZE);
+          }  
```

Comment:
How big is a typical `UPLOAD_CHUNK_SIZE`? I am trying to understand the kind of limit we are putting here.

Reply:
I think default is 64MB

Reply:
We have [CDPD-49309](https://jira.cloudera.com/browse/CDPD-49309) and [DOCS-15751](https://jira.cloudera.com/browse/DOCS-15751) related to the chunk size topic. But I am not sure these tickets track what's being discussed on this PR.

Reply:
If you think something should be documented, please open a DOCS ticket with the details and the releases/form-factors that are affected by the change, and we'll get those in. Thanks.

Reply:
@Harshg999 what do you think? We need this specific ozon size limit to be added to the docs right, it is not enough with the general chunk size documentation imo.

Reply:
Yes, we will document it for being Ozone specific.

Reply:
Please don't put internal JIRA links in upstream comments. Thanks!

Reply:
64MB upload limit seems to be extremely restrictive. Users frequently want to upload upwards of 1GB. I think we should get this commit in but open a channel with Ozone team on better understanding the challenges and finding a way to support large files.
cc: @subratanandi

Reply:
That is currently in the scoping phase for the Ozone team for them to support /append API and that way we fall back to the old upload flow like we have with other FS where this chunk size restriction is not an issue.

On the other hand, default value of the chunk size is 64MB. But we can always increase that to some higher value to incorporate a bigger file. We have tested till 400MB by increasing the chunk size but the main drawback is that it is very slow since it uses the /create API. We can discuss about the documentation works on this internally and for Ozone V1 we can go with this /create approach.

When Ozone teams supports /append, we can switch back for upload for better experience and scalability.